imm.yaml

```diff
@@ -3,29 +3,32 @@
     Id: 74e29f9b-6dca-4161-8fdd-b468a1eb185d
     Measures:
         -
-            Score: 0
-            Description: None
+            Score: 1
+            Description: Configured
```

Comment:
Not fully in line with our standards though. Does that matter?

Reply:
OK...so this is a fork of someone else's codebase, and as I understand it, it's a stopgap. I don't think the current plan is to fully take ownership of this. So given that, this is aligned to our standards as it ever will be.

So the deviations are not, in this case, an indication of the project being immature. They are because the deviations are appropriate. (And in general, you get a point for doing everything reasonable under the relevant rule. So in some cases, you can score a point for making a watertight case for why you're simply not going to do something. This is subtly different from rules where you can opt out, because you get a point for having done the necessary design work to establish that you don't need to do anything; this is not the same as cases where some particular rule just makes no sense for the IP in question.)

So my position is that since there's nothing more we want to do here, a full score is appropriate.

Reply:
I believe one of the concerns was to be able to merge in any changes made in the original repo, by the way. Rejigging everything to fully align with our styles would make this more or less impossible.

Reply:
However. on further reflection (and off-GitHub discussion with @jongeorge1 ) I've agreed that since we've drifted far enough that merging further changes looks challenging, we're really in some sort of limbo state. So setting this back to 0 as signpost that there's an issue here.